frontend/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run start:prod || npm run start:dev",
     "start:dev": "webpack-dev-server --inline --content-base public --history-api-fallback",
-    "start:prod": "webpack && node server.js",
+    "start:prod": "NODE_ENV=production webpack && node server.js",
```

Comment:
This command shouldn't be called directly. 

running `NODE_ENV=production npm start` will call `start:prod` and run them with `NODE_ENV=production`